docs/src/parallelization.md

```diff
@@ -155,3 +155,10 @@ To start Trixi in parallel with MPI, there are three options:
 For information on how to evaluate the parallel performance of Trixi, please
 have a look at the [Performance metrics of the `AnalysisCallback`](@ref)
 section, specifically at the descriptions of the performance index (PID).
+
+
+### Using error-based step size control with MPI
+If you use error-based step size control (see also the section on [error-based adaptive step sizes](https://trixi-framework.github.io/Trixi.jl/stable/tutorials/time_stepping/#Error-based-adaptive-step-sizes))
```

Comment:
Is there any way to create a consistent link instead of always linking to the latest stable version?

Reply:
Yes, it should be possible since we automatically preprocess  the tutorials [here](https://github.com/trixi-framework/Trixi.jl/blob/628aee3d0c31598e81ec9afaba9666ab051fad04/docs/literate/make.jl#L35) and add `(@id ...)`.
Therefore, we should be able to use a consistent link `(@ref ...)` like at the [tutorial starting page](https://github.com/trixi-framework/Trixi.jl/blob/628aee3d0c31598e81ec9afaba9666ab051fad04/docs/literate/src/files/index.jl#L102).

Reply:
If that's what  you mean and want  to  do.

Reply:
Yes, I think  so.

Reply:
Ok, I'll try it later. Thanks @bennibolm.

Reply:
I used the consistent link to the time stepping tutorial in 2d93808275bbe315ab81c94f23d49e7670e37425 (and removed a spurious bullet point in the tutorial introduction). Would it also be possible to have a link to the section [Error-based adaptive step sizes](https://github.com/trixi-framework/Trixi.jl/blob/main/docs/literate/src/files/time_stepping.jl#L21) instead of referring to the whole page? Otherwise we can also just keep it as it is.

Reply:
Theoretically, it would be possible to add  `[...](@id ...)` in the headline of this section and refer directly to this.
I don't know if that's wanted for the docs of Trixi.

Reply:
Nice, thanks. I did that in 3f7a62cb5b699e41d9800cbfe9c4cfa0bbc51520.